edb/server/mng_port/edgecon.pyx

```diff
@@ -2023,3 +2028,11 @@ cdef class EdgeConnection:
         msg.write_len_prefixed_bytes(b'RESTORE')
         self.write(msg.end_message())
         self.flush()
+
+
+@contextlib.contextmanager
+def timed(operation: str):
+    ts_start = time.time()
+    yield
+    ts_end = time.time()
+    logger.info("%s took %f seconds", operation, ts_end - ts_start)
```

Comment:
I'm torn on the log level here.  On one hand, telemetry is probably too verbose for `INFO`, but we don't want to enable `DEBUG` by default on those instances either.

Reply:
Yeah, I'm the same. Maybe we should do this as INFO but a different logger (e.g. `edb.server.telemetry`) which would be silenced unless RDS is used?

Reply:
We could add a new multi property to `cfg::Config` with a list of loggers to enable.

Reply:
I like the env var idea. Pretty easy to do and flexible enough for now.

Reply:
Do we want to have it as a DEBUG var?

Reply:
No, but we probably want some stats like p99, p90, p50, max.

Reply:
The p99 et al. reporting is done. Reports every 5 minutes per connection or at connection abort/close. Example:
```
INFO 60551 2020-06-23 22:13:20,870 edb.server: Opened a connection with ID 1; 1 open
INFO 60551 2020-06-23 22:14:14,262 edb.server: Query normalization stats: count=10, p99=0.0001; p90=0.0001; p50=0.0001; max=0.0001
INFO 60551 2020-06-23 22:14:14,262 edb.server: Query compilation stats: count=10, p99=0.3467; p90=0.3439; p50=0.1453; max=0.3470
INFO 60551 2020-06-23 22:14:14,262 edb.server: Closed a connection with ID 1; 0 open
INFO 60551 2020-06-23 22:14:14,263 edb.server: Killed a compiler-mng worker with PID 60654; 0 used, 4 in the pool; 5 spawned and 1 killed so far
```